tests/libfixmath/do-test.py

```diff
@@ -78,12 +78,16 @@ def main():
             abs_error = abs(res_locals['result'] - float(res_locals['expected']))
             res_locals['result'] = '{:.4f}'.format(res_locals['result'])
             if abs_error > ABS_ERROR_LIMIT:
-                print('{}: {} != {}, {:.4f} > {}'.format(res_locals['input'], res_locals['result'], res_locals['expected'],
-                                                         abs_error, ABS_ERROR_LIMIT))
+                print('{}: {} != {}, {:.4f} > {}'.format(
+                    res_locals['input'],
+                    res_locals['result'],
+                    res_locals['expected'],
+                    abs_error,
+                    ABS_ERROR_LIMIT))
                 errors += 1
-        except:
+        except Exception as exc:
             errors += 1
-            print('ERROR {}'.format(line))
+            print('ERROR: {}: {}'.format(line, exc))
```

Comment:
What PEP8 rule asks for an error to be printed? This also seems to contradict your changes in 335825e76ce24da877be02fb19d32576d0343841. Maybe a clearer commit message would already help.

Reply:
If you have concerns on this change, please review #12687 where this commit comes from. Thanks!

Reply:
Sorry, wasn't aware this was part of #12687.

Reply:
Why does a global style fix PR depend on an enhancement for a single package btw? Oo

Reply:
Well it doesn't depend, its just I was aware of @aabadie changes on libfixmath, so instead of fixing them I'm willing to wait for it to be fixed in #12687 :), otherwise it would need rebase at some point from one of us since he is also changing the test, renaming the folder, etc.